lib/crowdin-api/client/client.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+# The Crowdin::Client library is used for interactions with a crowdin.com website.
+#
+# == Example
+#
+#   require 'crowdin-api'
+#
+#   crowdin = Crowdin::Client.new(api_key: API_KEY)
+#
+module Crowdin
+  class Client
+    # A wrapper and interface to the Crowdin api. Please visit the Crowdin developers
+    # site for a full explanation of what each of the Crowdin api methods
+    # expect and perform.
+    #
+    # https://support.crowdin.com/api/v2/
+    #
+    include API::Languages
+    include API::Projects
+    include API::SourceFiles
+    include API::Storages
+    include API::TranslationStatus
+    include API::Translations
```

Comment:
When I was thinking about Ruby API for this gem I thought about one idea about scoping. There are some API endpoints that are scoped inside others. For example `Translations` are scoped to projects. So I thought that it would be good if the resources were scoped the same way, like:
```rb
  Crowdin::API::Projects.new(project_id: 'XXXX').upload_translations('UA')
```

I see you did it differently and it's fine too, but I wanted to leave it here just for history. Maybe you'll like this idea and consider it.

Reply:
I'll take note about this as really good point